src/test/java/teammates/test/cases/ui/InstructorEditStudentFeedbackPageActionTest.java

```diff
@@ -13,24 +13,24 @@
 import teammates.common.exception.UnauthorizedAccessException;
 import teammates.common.util.Const;
 import teammates.ui.controller.InstructorEditStudentFeedbackPageAction;
-import teammates.ui.controller.InstructorFeedbackPreviewAsStudentAction;
 import teammates.ui.controller.ShowPageResult;
 
-public class InstructorEditStudentFeedbackPageActionTest extends
-        BaseActionTest {
-    private final DataBundle dataBundle = getTypicalDataBundle();
+public class InstructorEditStudentFeedbackPageActionTest extends BaseActionTest {
+    private static DataBundle dataBundle;
     
     @BeforeClass
     public static void classSetUp() throws Exception {
         printTestClassHeader();
-        removeAndRestoreTypicalDataInDatastore();
+        dataBundle = loadDataBundle("/InstructorEditStudentFeedbackPageTest.json");
+        removeAndRestoreDatastoreFromJson("/InstructorEditStudentFeedbackPageTest.json");
+        
         uri = Const.ActionURIs.INSTRUCTOR_EDIT_STUDENT_FEEDBACK_PAGE;
     }
     
     @Test
-    public void testExecuteAndPostProcess() throws Exception{
-        InstructorAttributes instructor = dataBundle.instructors.get("instructor1OfCourse1");
-        InstructorAttributes instructorHelper = dataBundle.instructors.get("helperOfCourse1");
+    public void testExecuteAndPostProcess() throws Exception {
+        InstructorAttributes instructor = dataBundle.instructors.get("IESFPTCourseinstr");
+        InstructorAttributes instructorHelper = dataBundle.instructors.get("IESFPTCoursehelper1");
```

Comment:
Migrated everything to using IESFPT data instead of typical bundle so that it tests all the different settings from testExecuteAndPostProcess()